src/redux/reducers/index.js

```diff
@@ -21,4 +22,5 @@ export default combineReducers({
   communities,
   user,
   cache: cacheReducer,
+  tooltips: tooltipsReducer,
```

Comment:
@aliseyalvi  perhaps renaming `tooltipsReducer` / actions to `walkthroughReducer`/action will make more self elaborative, what do you think?